server.py

```diff
@@ -427,52 +430,43 @@ def StartServer(port, syncer):
 
 
 def main():
-  import argparse
-  parser = argparse.ArgumentParser(description="N.E.BRE-8 control server")
-  parser.add_argument('--port', type=int, default=8000, help='Port to run on')
-  parser.add_argument('--fake', dest='fake', action='store_true')
-  parser.add_argument('--nofake', dest='fake', action='store_false')
-  parser.add_argument('--logfile',
-                      default="",
-                      help='File to log to. If empty, does not log to a file')
-  parser.add_argument('--logtostderr', dest='logtostderr', action='store_true')
-  parser.add_argument('--loglevel',
-                      default='info',
-                      help='Log level (debug, info, warning, error)')
-  parser.add_argument('--frontend',
-                      default='http://nebree8.appspot.com',
-                      help='Frontend server to sync with, if any')
-  parser.add_argument('--fe_poll_freq',
-                      type=int,
-                      default=5,
-                      help='Frontend polling frequency in seconds')
-  parser.set_defaults(fake=False, logtostderr=True)
-  args = parser.parse_args()
+  import sys
+  FLAGS(sys.argv)
 
   # Set up logging
   rootLogger = logging.getLogger()
-  rootLogger.setLevel(getattr(logging, args.loglevel.upper()))
-  if args.logfile:
-    rootLogger.addHandler(logging.FileHandler(args.logfile))
-  if args.logtostderr:
+  rootLogger.setLevel(getattr(logging, FLAGS.loglevel.upper()))
```

Comment:
The old implementation had a default value for loglevel. Where are you setting the default for FLAGS.loglevel? Same for other flags that used to have defaults.